apps/www/_blog/2022-10-03-postgres-wasm.mdx

```diff
@@ -0,0 +1,253 @@
+---
+title: Postgres WASM by Snaplet and Supabase
```

Comment:
@kiwicopple  Should we add Open Source to the title? So: Open source Postgres WASM by Snaplet and Supabase

Reply:
I think better as it is - this title is enough to pique the interest and they will see it's open source from the first paragraph